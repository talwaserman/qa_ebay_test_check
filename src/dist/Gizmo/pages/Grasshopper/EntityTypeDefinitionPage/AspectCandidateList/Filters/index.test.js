"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var _index = _interopRequireDefault(require("./index"));

var initialValues = ['added'];
it('Gizmo grasshopper - Filters in candidate list', function () {
  var tree = _reactTestRenderer.default.create( /*#__PURE__*/_react.default.createElement(_index.default, {
    initialValues: initialValues
  })).toJSON();

  expect(tree).toMatchSnapshot();
});