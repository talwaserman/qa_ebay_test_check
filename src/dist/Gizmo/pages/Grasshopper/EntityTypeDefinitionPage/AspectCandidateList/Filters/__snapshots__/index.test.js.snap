// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gizmo grasshopper - Filters in candidate list 1`] = `
<div
  className="filters-wrapper"
>
  <button
    className="ant-btn ant-btn-sm"
    onClick={[Function]}
    onMouseDown={[Function]}
    onTouchStart={[Function]}
    type="button"
  >
    <span>
      Filters
    </span>
  </button>
</div>
`;
